chapters/en/slides/module1/module1_01.Rmd

```diff
@@ -61,242 +64,122 @@ as we will see later.
-<td>8.0</td>
-<td>6.95</td>
-<td>8.0</td>
-<td>8.14</td>
-<td>8.0</td>
-<td>6.77</td>
-<td>8.0</td>
-<td>5.76
-</td></tr>
-<tr>
-<td>13.0</td>
-<td>7.58</td>
-<td>13.0</td>
-<td>8.74</td>
-<td>13.0</td>
-<td>12.74</td>
-<td>8.0</td>
-<td>7.71
-</td></tr>
-<tr>
-<td>9.0</td>
-<td>8.81</td>
-<td>9.0</td>
-<td>8.77</td>
-<td>9.0</td>
-<td>7.11</td>
-<td>8.0</td>
-<td>8.84
-</td></tr>
-<tr>
-<td>11.0</td>
-<td>8.33</td>
-<td>11.0</td>
-<td>9.26</td>
-<td>11.0</td>
-<td>7.81</td>
-<td>8.0</td>
-<td>8.47
-</td></tr>
-<tr>
-<td>14.0</td>
-<td>9.96</td>
-<td>14.0</td>
-<td>8.10</td>
-<td>14.0</td>
-<td>8.84</td>
-<td>8.0</td>
-<td>7.04
-</td></tr>
-<tr>
-<td>6.0</td>
-<td>7.24</td>
-<td>6.0</td>
-<td>6.13</td>
-<td>6.0</td>
-<td>6.08</td>
-<td>8.0</td>
-<td>5.25
-</td></tr>
-<tr>
-<td>4.0</td>
-<td>4.26</td>
-<td>4.0</td>
-<td>3.10</td>
-<td>4.0</td>
-<td>5.39</td>
-<td>19.0</td>
-<td>12.50
-</td></tr>
-<tr>
-<td>12.0</td>
-<td>10.84</td>
-<td>12.0</td>
-<td>9.13</td>
-<td>12.0</td>
-<td>8.15</td>
-<td>8.0</td>
-<td>5.56
-</td></tr>
-<tr>
-<td>7.0</td>
-<td>4.82</td>
-<td>7.0</td>
-<td>7.26</td>
-<td>7.0</td>
-<td>6.42</td>
-<td>8.0</td>
-<td>7.91
-</td></tr>
-<tr>
-<td>5.0</td>
-<td>5.68</td>
-<td>5.0</td>
-<td>4.74</td>
-<td>5.0</td>
-<td>5.73</td>
-<td>8.0</td>
-<td>6.89
-</td></tr></tbody></table>
+```{python echo=FALSE, results='asis'}
+# This is the code that generates the table below,
+# and setting results to `'asis'` ensures it is rendered as HTML
+# instead of being wrapped in an output code block
+import altair as alt
+from vega_datasets import data
+
+anscombe = data.anscombe()#.query('Series != "IV"')
+anscombe['Set'] = anscombe['Series'].map({'I': 'A', 'II': 'B', 'III': 'C', 'IV': 'D'})
+anscombe = anscombe.drop(columns='Series')
+anscombe.loc[anscombe['Y'] == anscombe['Y'].max(), 'Y'] = 8.5
+
+for set_name, df in anscombe.groupby('Set'):
+    df = df.drop(columns='Set')
+    df.columns = pd.MultiIndex.from_tuples([(set_name, 'X'), (set_name, 'Y')])
+    print(
+        df
+        .to_html(index=False)
+        .replace('class="dataframe"', 'style="width:70px; float:left; margin-right:40px"')
+        .replace('<thead>',  '<thead style="text-align: center">'))
+```
 
 Notes:
-Think about it, numbers have been around for about 5,000 years.
-Visual systems, on the other hand, have undergone refinement for over
-500,000,000 years by training with a lethal, evolutionary cost function.
-While we need to train ourselves to recognize structure in numerical
-data, we have evolved to instinctively recognize visual patterns and to
-accurately judge properties such as colors and distances between
-objects. Drawing upon this complex build-in apparatus, we can arrive at
-conclusion much faster by glancing at visual representations of data
+Think about it,
+numbers have only been around for about 5,000 years.
+Visual systems on the other hand,
+have undergone refinement for over 500,000,000 years
+by training with a lethal, evolutionary cost function.
```

Comment:
Non-bio majors are going to not understand "training with a lethal, evolutionary cost function" can we simply/reduce jargon here? I like it, but for a general audience this could confuse.

Reply:
I was thinking "cost function" would be OK given that they just had the ML course, but I mayeb not all students have taken that? I rephrased.

Reply:
No, not all students will have taken that. The only pre-req for this course is the programming course.